snuba/datasets/entities/events.py

```diff
@@ -198,9 +198,7 @@ def get_query_processors(self) -> Sequence[QueryProcessor]:
             ),
             TagsExpanderProcessor(),
             BasicFunctionsProcessor(),
-            HandledFunctionsProcessor(
-                "exception_stacks.mechanism_handled", self.get_data_model()
-            ),
+            HandledFunctionsProcessor("exception_stacks.mechanism_handled", self),
```

Comment:
See comment below. I don't think we should need to pass the data model to these processor at instantiation time at all.